### ISSUE TITLE
[Linux] NFSW doesn't launch in clean wineprefix

### ISSUE THREAD

**Issue by mradermaxlol** [SoapboxRaceWorld/GameLauncher_NFSW#139]
- **Launcher Version:** 2.1.7.6
- **Operating system:** Arch Linux x86_64
- **Antivirus/Firewall (if any):** no
- **Wine/Mono Version (if any):** wine-6.2, wine-mono-x86-6.0.0

Game doesn't start using vanilla wine-6.2 on Arch x86_64 with testing+multilib repos. My setup:
1) a clean x86_64 (with WoW64) wineprefix was created using `env WINEPREFIX="/home/peppermint/.wine_nfsw/"`, no messing around with anything
2) gamefiles are in wineprefix's C:\NFS (already downloaded the files before (from the launcher ofc) and kept moving them around for testing)
3) dxvk/gallium nine/anything else disabled
4) `mesa_glthread=true` overriden by my /etc/environment
5) launcher resides in ~/nfsw, I `cd`'d to the directory and then launched the executable (LinuxCompat files were present btw)

- Console output collected using `env WINEPREFIX="/home/peppermint/.wine_nfsw/" wine GameLauncher.exe >> log.txt 2>&1`: [log.txt](https://github.com/SoapboxRaceWorld/GameLauncher_NFSW/files/6056486/log.txt)
- Crash message experienced right after opening the game window (before the loading process):
![Screenshot_20210228_120418](https://user-images.githubusercontent.com/11138966/109413536-05ec0380-79bf-11eb-9416-d533782c6d91.png)
- Wine backtrace for the crash: 
[backtrace.txt](https://github.com/SoapboxRaceWorld/GameLauncher_NFSW/files/6056490/backtrace.txt)
- The error launcher shows right after that:
![Screenshot_20210228_120502](https://user-images.githubusercontent.com/11138966/109413561-24ea9580-79bf-11eb-9288-58c0bfa8c5d9.png)

Additional info:
- Wine stuff:
![Screenshot_20210228_120627](https://user-images.githubusercontent.com/11138966/109413573-32078480-79bf-11eb-9dd4-ce6cf887f379.png)
- multilib packages installed:
[pkgs.txt](https://github.com/SoapboxRaceWorld/GameLauncher_NFSW/files/6056491/pkgs.txt)
- hw/system info probably doesn't matter but just in case:
![Screenshot_20210228_122109](https://user-images.githubusercontent.com/11138966/109413635-7c890100-79bf-11eb-94ec-0cd258c73ad8.png)

It would be nice if instructions for making the game playable using vanilla/staging Wine were provided in the readme (or at least things like "you need to have x/y/z installed in your prefix") as
1) not everyone fancies using Lutris
2) this launcher's stuff on Lutris is outdated anyway

**Comment by mradermaxlol** [comment]
I also found a separate issue yesterday: if you don't manually set the path to gamefiles to someplace in your prefix in `Settings.ini` on Linux then the launcher will throw errors all the time, won't even allow you to change the installation path/login/actually download anything. It has something to do with fs permissions, something like "wine doesn't allow the application to write directly to some unixpath".
**TL;DR** you have to set your `InstallationDirectory = C:\some_path_over_there` in Settings.ini to make the launcher work.
